### DISCUSSION TITLE
Opentrace tracer

### DISCUSSION THREAD

**Discussion by mdaliyan** [JanusGraph/janusgraph#2403] [Ideas]
We're using openTracing in our stack, and since Cassandra supports it through a plugin, is there any way we can use tracing on JG?

**Comment by farodin91** [comment]
It would also require to add OpenTracing to tinkerpop. I don't think this is already done.

**Reply by cdegroc** [reply]
I've opened a ticket for this: https://issues.apache.org/jira/browse/TINKERPOP-2523
Feel free to comment there :)

**Comment by alexvanboxel** [comment]
For all new efforts, I would advise to skip OpenTracing and go directly to OpenTelemetry ( it a merger of OpenTracing and OpenCensus) and will soon be GA for tracing.
